### DISCUSSION TITLE
Stream Deck+?

### DISCUSSION THREAD

**Discussion by KrysDisco** [nguyenquyhy/Flight-Tracker-StreamDeck#365] [Ideas]
Hi,
Love the plugin for Strean Deck. Do you plan to upgrade it for the Stream Deck+ with rotary know?
Would really love that on buttons. THanks in advance!
Krys

**Comment by nguyenquyhy** [comment]
I have already bought one, but Elgato is still keeping the new SDK private, so I can't really do any work on the dial yet.

Definitely will try to do something for the new dial once I can.

**Reply by KrysDisco** [reply]
Appreciate it thank you. Finger crossed that they release the SDK soon.
Good job anyway.

**Reply by KrysDisco** [reply]
Hello, And happy new year.
I am mistaken or they updated the web page with SDK for the Stream Deck +?
https://developer.elgato.com/documentation/stream-deck/sdk/overview/#stream-deck
Have a nice day.
Krys

**Reply by nguyenquyhy** [reply]
Oh I have added StreamDeck+ support 2 releases ago :). Try it out!

**Reply by nguyenquyhy** [reply]
What is already supported:
- Preset functions
- COM/NAV numpad

What is not there yet:
- Generic functions

**Reply by vcsandstorm** [reply]
I hope the Dial Generic Function will be coming soon! Love your work! @nguyenquyhy

**Comment by KrysDisco** [comment]
I checked quickly, it is super promising.
It will be super cool with all dial functions like the AP Speed control,
etc.
Good job!

Le mar. 3 janv. 2023 à 17:50, Nguyễn Quý Hy ***@***.***> a
écrit :

**Reply by YodaVonBeck** [reply]
I was also looking for a way to set the speed with the Stream Deck + model.
Its the 1 thing that is missing from my minimal point of view to be able to fly an airliners like the 320, without having to use my mouse to turn the knobs inside the aircraft.

Unless there is another way around this :)

**Reply by nguyenquyhy** [reply]
A320 will need a new Generic Dial Button, so you can tie the rotate events with MobiFlight to update LVar. I am halfway into adding the button but did not have time to finish that you.

**Comment by johans1234** [comment]
Hi,
I am a new user of the Streamdeck + and started to figure out the capabilities of the Flight Tracker Plugin. I am of cource most interested in the new stuff I can do with the rotary buttons. HEADING, ALTITUDE, VERTICAL SPEED are working very well but the VOR1 and VOR2 functions are not working for me. Not in FS 2020 nor in FS 2024. Is there something to consider to use this functions?

regards Sascha

PS: again, many thanks for the great job !!!!!!!!!!!!!!!!!!!!!!!!!!!!

**Reply by nguyenquyhy** [reply]
You are right, VOR1 and 2 are broken. I'll look into it this weekend.

**Reply by nguyenquyhy** [reply]
Issue to track https://github.com/nguyenquyhy/Flight-Tracker-StreamDeck/issues/432
